GooglePlayInstant/Tests/Editor/QuickDeploy/ConfigurationTest.cs

```diff
@@ -38,10 +38,7 @@ public void TestSavingConfigOnCreateBundleWithString()
         {
             var quickDeployConfig = new QuickDeployConfig
             {
-                AssetBundleFileName = "testbundle",
```

Comment:
You didn't delete assetBundleFileName above? Should you delete it or add the test back here?

Reply:
Not sure what you mean here. Is the confusion from the strange looking diff? A note that while it looks like I deleted AssetBundleFileName, I didn't. I replaced it with the same line, but no comma at the end, because I still wanted to test the string's persistence.